File: ccc/2019/j1.java
// https://dmoj.ca/problem/ccc19j1
//CCC '19 J1 - Winning Score

import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
		int a3 = s.nextInt();
		int a2 = s.nextInt();
		int a1 = s.nextInt();
		int b3 = s.nextInt();
		int b2 = s.nextInt();
		int b1 = s.nextInt();
		if ((a3*3 + a2*2 + a1*1) > (b3*3 + b2*2 + b1*1)) {
	    	System.out.println("A");
		} else if ((a3*3 + a2*2 + a1*1) == (b3*3 + b2*2 + b1*1)) {
			System.out.println("T");
		} else {
			System.out.println("B");
		}
		s.close();
    }
}

File: ccc/2016/j1.py
#https://dmoj.ca/problem/ccc16j1
#CCC '16 J1 - Tournament Selection

a = input()
b = input()
c = input()
d = input()
e = input()
f = input()

w = 0
l = 0

if a == "W":
    w += 1
if b == "W":
    w += 1
if c == "W":
    w += 1
if d == "W":
    w += 1
if e == "W":
    w += 1
if f == "W":
    w += 1

if w == 0:
    print("-1")
elif w <= 2:
    print("3")
elif w <= 4:
    print("2")
else:
    print("1")
